tests/unit/src/test/scala/tests/OnTypeFormattingSuite.scala

```diff
@@ -168,7 +168,7 @@ class OnTypeFormattingSuite extends BaseLspSuite("onTypeFormatting") {
        |  | a multiline
        |  | string
        |  '''.stripMargin
-       |
+       |  
```

Comment:
Ha, I have the same thing and my vim setup removes empty spaces at the end of a line with I format. This pr that @gabro did is a clever way to get around it and it also has the benefit of being more explicit that we want an `indent` there. You can probably utilize the same idea.